docker-compose-mastodon-update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+git stash
+git pull
```

Comment:
I'm wary of using `git pull` here because we'd like to encourage admins to run on tagged versions of mastodon, instead of whatever happens to be on master at the time they upgrade. What do you think of changing this to ask for a tagged version instead?